build/patches/Remove-EV-certificates.patch

```diff
@@ -7,42 +7,36 @@ certificates. The web of trust is considered a failure in itself, so
 do not give users a false sense of extra security with EV certs.
 Instead, let them appear just like regular ones.
 ---
- net/cert/ev_root_ca_metadata.cc | 19 ++++++++++++++++++-
+ net/cert/ev_root_ca_metadata.cc | 20 +++++++++++++++++++-
  net/cert/ev_root_ca_metadata.h  |  4 +---
- 2 files changed, 19 insertions(+), 4 deletions(-)
+ 2 files changed, 20 insertions(+), 4 deletions(-)
 
 diff --git a/net/cert/ev_root_ca_metadata.cc b/net/cert/ev_root_ca_metadata.cc
 --- a/net/cert/ev_root_ca_metadata.cc
 +++ b/net/cert/ev_root_ca_metadata.cc
-@@ -44,6 +44,16 @@ static const EVMetadata kEvRootCaMetadata[] = {
-     // AC Camerfirma S.A. Chambers of Commerce Root - 2008
-     // https://www.camerfirma.com
-     {
-+  // need some dummy thing to make compiler happy, because
-+  // arraysize() is implemented as a convoluted template rather than
-+  // the traditional sizeof(x)/sizeof(*x)
-+  { { { 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
-+        0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff } },
+@@ -41,7 +41,20 @@ struct EVMetadata {
+   const base::StringPiece policy_oids[kMaxOIDsPerCA];
+ };
+ 
+-#include "net/data/ssl/ev_roots/chrome-ev-root-store-inc.cc"
++
++static const EVMetadata kEvRootCaMetadata[] = {
```

Comment:
it doesn't seem to be used anymore.
in any case `PLATFORM_USES_CHROMIUM_EV_METADATA` is not defined, so the code doesn't go through here

Reply:
I will leave it in case it gets re-enabled, as we do for other patches too.